### PR TITLE
NMS-13811: Troubleshooting flows: basic

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13811

### CODE REVIEW

**Review by swachter — CHANGES_REQUESTED** [review]
The Grafana-Templating issue needs some overhaul.

File: docs/modules/operation/pages/flows/troubleshooting.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+A "template is missing" error message in Grafana can occur if, the first time you set up flows, you start OpenNMS before you start Elasticsearch.
+If you start OpenNMS before Elasticsearch, the required template will not be available, and Elasticsearch does not know what to do with the flows documents.
+This template is installed once, the first time you connect to Elasticsearch.
+
+To fix this issue, follow these steps:
+
+. Stop OpenNMS.
+. Stop Elasticseach and Elasticsearch data.
+. Start Elasticsearch.
+. Start OpenNMS.
```
[the remark this answers is not in the document]
Comment (Bonrob2) [reply docs/modules/operation/pages/flows/troubleshooting.adoc:69]: Thanks. I will remove this section.

**Review by swachter — APPROVED** [review]
LGTM

File: docs/modules/operation/pages/flows/troubleshooting.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+In a scenario where you see no data or incorrect data, you should view the state and parameters of telemetry listeners, and whether they are processing data and information with the following command: `opennms:telemetry-listeners`
+
+== Check Elasticsearch persistence
+
+If you store flows in Elasticsearch, you can use Kibana to check if flow documents (raw and/or aggregated) are written to Elasticsearch.
+You must know your endpoint address and API key.
+
+Run the following curl command:
+
+[source, curl]
```
Comment (mmahacek) [inline docs/modules/operation/pages/flows/troubleshooting.adoc:32]:
```suggestion
[source, console]
```

File: docs/modules/operation/pages/flows/troubleshooting.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+----
+curl "https://Your Endpoint-es.logit.io/_cat/indices?v&apikey=Your-API-Key"
```
Comment (mmahacek) [inline docs/modules/operation/pages/flows/troubleshooting.adoc:34]: I'd use a more properly formatted URL example
```suggestion
curl "https://your-es-server.hostname:9200/_cat/indices?v&apikey=Your-API-Key"
```

**Review by mmahacek — COMMENTED** [review]
a couple minor suggestions, but nothing I'd count as blocking

**Comment by Bonrob2** [comment]
@mmahacek  I updated based on your feedback, but lost the approval for the PR. Could you approve so I can merge?

**Review by mmahacek — APPROVED** [review]

**Pull request merged**